analyze.js

```diff
@@ -142,7 +142,10 @@ async function getProjectId(config, client) {
     if (matchedProjectIds.length == 1) {
       return matchedProjectIds[0]
     } else if (matchedProjectIds.length == 0) {
-      throw new Error(`No projects with the name '${config.projectName}'.`)
+      core.info(`No projects found with the name '${config.projectName}'. Creating the project...`)
+      const createdProject = await client.createSrmProject(config)
+      core.info(`Created project '${createdProject.name}' (projectId = ${createdProject.id})`)
+      return createdProject.id
```

Comment:
It would be good to also guard this behavior with a setting to prevent accidental project creation in case of a typo, something like `auto-create-project` which defaults to `false`